extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/internal/type/MetadataUtils.java

```diff
@@ -290,6 +291,41 @@ public static String getItemType(HmDatapoint dp) {
                     && !channelType.equals(CHANNEL_TYPE_AKKU)) {
                 return ITEM_TYPE_DIMMER;
             } else {
+                // determine QuantityType
+                switch (dp.getUnit()) {
```

Comment:
does this add support for UoM also on all the dynamically created channels?

Reply:
Exactly.